pkg/cli/admin/inspect/inspect.go

```diff
@@ -100,6 +103,8 @@ func NewCmdInspect(streams genericclioptions.IOStreams, parentCommandPath string
 
 	cmd.Flags().StringVar(&o.destDir, "dest-dir", o.destDir, "Root directory used for storing all gathered cluster operator data. Defaults to $(PWD)/inspect.local.<rand>")
 	cmd.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", o.allNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
+	cmd.Flags().Int64Var(&o.since, "sinceseconds", o.since, "An RFC3339 timestamp from which to show logs. If this value precedes the time a pod was started, only logs since the pod start will be returned.")
+	cmd.Flags().DurationVar(&o.sincetime, "sincetime", o.sincetime, "Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of sinceTime or sinceseconds may be used.")
```

Comment:
The names are still wrong, please take a look at `oc logs -h` output and apply exactly the same names here:
```
--since=0s: Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of since-time / since may be used.
--since-time='': Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of since-time / since may be used.
```

Reply:
I have made the changes , should i add the hyphens before since-time also